usmqe/api/tendrlapi/authentication.py

```diff
@@ -0,0 +1,60 @@
+"""
+Tendrl REST API.
+"""
+
+import requests
+
+import pytest
+
+import json
+
+from usmqe.api.tendrlapi.common import TendrlApi
+
+LOGGER = pytest.get_logger("tendrlapi.authentication", module=True)
+
+
+class Authentication(TendrlApi):
+    """ Main class for interact with REST API - authentication."""
+
+    def __init__(self):
+        self.username = None
+        self.access_token = None
+
+    def login(self, username, password, asserts_in=None):
+        """ Login user.
+
+        Name:        "login",
+        Method:      "GET",
+        Pattern:     "login",
+
+        Args:
+            username: name of user that is going logged in
+            password: password for username
+            asserts_in: assert values for this call and this method
+        """
+        pattern = "login"
```

Comment:
While this looks weird at first sight, similar code structure is used in the rest of the TendrlApi methods, so I guess it's ok with this in mind.